### PR TITLE
Expose conf folder in the JDK repository

### PR DESCRIPTION
**Pull request by menny**
the conf folder holds the security policy files.

### CODE REVIEW

File: src/main/tools/jdk.BUILD [hunk 1]
```diff
@@ -1,6 +1,6 @@
-package(default_visibility = ["//visibility:public"])
+load("@rules_java//java:defs.bzl", "java_import", "java_runtime")
 
-load("@rules_java//java:defs.bzl", "java_runtime", "java_import")
+package(default_visibility = ["//visibility:public"])
```
Comment (menny) [inline src/main/tools/jdk.BUILD:3]: These two lines will make `buildifier` happy.

File: src/main/tools/jdk.BUILD [hunk 2]
```diff
@@ -181,6 +181,12 @@ filegroup(
     ),
 )
 
+#This folder holds security policies
+filegroup(
+    name = "jdk-conf",
+    srcs = glob(["conf/**"]),
```
Comment (menny) [inline src/main/tools/jdk.BUILD:187]: The `conf` folder holds security policy files (`limited` and `unlimited`). We use a Java mock-server that in tests that uses an untrusted cert, so we rely on `unlimited` policy to allow this to work.

File: src/main/tools/jdk.BUILD (hunk 1 above)
Comment (menny) [inline src/main/tools/jdk.BUILD:3]: These two lines will make buildifier happy.

**Comment by aiuto** [comment]
Liam: Can you take a look at this?

File: src/main/tools/jdk.BUILD (hunk 2 above)
Comment (menny) [reply src/main/tools/jdk.BUILD:187]: we use Corretto JDK, and it has about 20 files under that folder (and its sub-folders) around 100kb in size.

**Comment by philwo** [comment]
I looked into this now (sorry for the delay 😞 ), but I think most targets in this BUILD file are deprecated.

Unfortunately I don't understand how to use the recommended replacement `@bazel_tools//tools/jdk:current_java_runtime` instead of these filegroups.

@cushon @comius Can you help?

**Review by comius — APPROVED** [review]

**Comment by comius** [comment]
@philwo: the changes look fine to me, the particular targets (jdk, java-includes) in jdk.BUILD are used, and I don't think you  can extend them in tools/jdk/BUILD.

I checked JDK11 and JDK14, and they have conf directory, also with other configuration files, not just policy. JDK8 does not yet have a conf folder. Also our remote JDKs have the conf directory.

@menny Can you rebase and resolve the conflicts?

**Comment by comius** [comment]
closes https://github.com/bazelbuild/bazel/issues/9189

**Comment by menny** [comment]
@comius done. Thank you.

**Pull request closed without merging**